hooks/pre-push

```diff
@@ -50,7 +50,7 @@ bash scripts/run_linter.sh
 check_test_status "Linter"
 
 echo "Running karma tests"
-bash scripts/run_karma_tests.sh --browsers PhantomJS
```

Comment:
The phantomJS browser was here in case the actual browser cannot run (for example when in a SSH session). Removing may cause errors in such cases.